docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Why not have this rule in a top dir Makefile?

Reply:
I have not experimented with this but without this Makefile here it will probably complain during the docs building.

Reply:
I have mine at the root dir, no issues at all